README.md

```diff
@@ -727,13 +727,13 @@ For IDE development we recommend the [IntelliJ IDEA Community Edition](https://w
 can directly import the Gradle build files. After cloning the project's source code recursively, simply run IDEA and use
 the following steps to import the project.
 
-1. From the wizard dialog: Select *Import Project*.
+1. From the welcome dialog: Select `Open`.
```

Comment:
We should probably say "*Welcome* dialog" here for consistency with "*Open Project* dialog" below.

Reply:
Change was applied.